c7n/resources/dms.py

```diff
@@ -112,6 +113,12 @@ def process_resource_set(self, client, resources):
             r['Tags'] = tags
 
 
+@ReplicationInstance.filter_registry.register('kms-key-alias')
```

Comment:
since its more generic now we should just have this as `kms-key` since its not specific to the alias alone.